src/measure_theory/integral/interval_integral.lean

```diff
@@ -383,25 +383,28 @@ lemma sum (s : finset ι) {f : ι → ℝ → E} (h : ∀ i ∈ s, interval_inte
   interval_integrable (∑ i in s, f i) μ a b :=
 ⟨integrable_finset_sum' s (λ i hi, (h i hi).1), integrable_finset_sum' s (λ i hi, (h i hi).2)⟩
 
-lemma mul_continuous_on {f g : ℝ → ℝ}
+lemma mul_continuous_on {A : Type*} [normed_ring A] [second_countable_topology A] {f g : ℝ → A}
```

Comment:
Can you add `{A : Type*}` at the top of the file between `𝕜` and `E`?

Reply:
I'd be happy to, but I'm curious why this is regarded as beneficial, could you explain? (This is a genuine question, honestly no sarcasm is intended).

Reply:
Mainly it's beneficial because it establishes a convention throughout the file rather than ending up with an ever-growing list of variable names. The other benefit is that aesthetically it's nice to have all the type arguments before the typeclass arguments.

Reply:
```suggestion
lemma mul_continuous_on [normed_ring A] [second_countable_topology A] {f g : ℝ → A}
```

Reply:
(and below)